api/CMakeLists.txt

```diff
@@ -53,6 +54,7 @@ target_sources(api
     hinted_handoff.cc
     lsa.cc
     messaging_service.cc
+	raft.cc
```

Comment:
could you use spaces instead of tab here?

Reply:
Okay!

Reply:
Done.